roles/database/tasks/main.yml

```diff
@@ -10,3 +10,7 @@
       - name: "{{ database_user }}"
         priv: "{{ database_name }}.*:ALL"
         password: "{{ database_password }}"
+      - name: "{{ database_metabase_user }}"
+        priv: "*.*:REQUIRESSL/{{ database_name }}.people:SELECT(id, uuid, created_at, updated_at, is_admin, locale, preferences, active_days_count, last_page_load_date, test_group_number, username, reset_password_sent_at, remember_created_at, sign_in_count, current_sign_in_at, last_sign_in_at, given_name, family_name, display_name, phone_number, description, image_updated_at)/{{ database_name }}.categories:SELECT/{{ database_name }}.comments:SELECT/{{ database_name }}.conversations:SELECT/{{ database_name }}.follower_relationships:SELECT/{{ database_name }}.listings:SELECT/{{ database_name }}.locations:SELECT/{{ database_name }}.messages:SELECT/{{ database_name }}.participations:SELECT/{{ database_name }}.shipping_addresses:SELECT/{{ database_name }}.stripe_payments:SELECT/{{ database_name }}.testimonials:SELECT/{{ database_name }}.transactions:SELECT/{{ database_name }}.transaction_processes:SELECT/{{ database_name }}.transaction_transitions:SELECT"
+        password: "{{ database_metabase_password }}"
+        host: "static.42.151.203.116.clients.your-server.de"
```

Comment:
This is the host name that finally reaches the MySQL DB, using `metabase.coopdevs.org` doesn't work.

Reply:
have you asked Hetzner's customer support? given this domain name, we might as well use the plain IP, and that would be fine IMO.

Reply:
I didn't, I don't see it as a priority.
Would be nice to give it a try with the plain IP but it's not like 5 minutes. I would leave it as it is.